DEVELOPMENT.md

```diff
@@ -345,3 +351,26 @@ To access the Splunk web interface from outside of the cluster, the following st
    export KIND_CLUSTER_NAME=$(kind -q get clusters)
    ```
    **NOTE**: In case you have more than one kind cluster running, manually set the proper one
+
+1. Deploying the operator (e.g. using the `hack/quick-install.sh` script), the `Deployment` doesn't reach `Ready` state:
```

Comment:
You shouldn't use Latin abbreviations (from Neal) - spell out what you mean instead.

```suggestion
1. Deploying the operator (for example: using the `hack/quick-install.sh` script), the `Deployment` doesn't reach `Ready` state:
```

Reply:
Done!